.agent/specs/100.md

```
@@ -0,0 +1,264 @@
+
+### Repository 인터페이스 변경
+
+```java
+// WorkflowDefinitionRepository.java 에 추가
+boolean existsByDomainPackVersionIdAndPolicyRef(Long versionId, String policyCode);
+```
+
+### JPA 구현 — JSONB 역참조 쿼리
+
+graphJson 구조:
+```json
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)